src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -171,6 +172,28 @@ public function getListIdsAction(int $productId): JsonResponse
         return $this->json($data);
     }
 
+    /**
+     * @AdminSecurity("is_granted('delete', request.get('_legacy_controller'))")
+     *
+     * @param int $combinationId
+     *
+     * @return JsonResponse
+     */
+    public function removeAction(int $combinationId): JsonResponse
+    {
+        try {
+            $this->getCommandBus()->handle(new RemoveCombinationCommand($combinationId));
+        } catch (Exception $e) {
+            return $this->json([
+                'error' => $this->getErrorMessageForException($e, $this->getErrorMessages($e)),
+            ], Response::HTTP_BAD_REQUEST);
+        }
+
+        return $this->json([
```

Comment:
Should we return the HTTP code 204 ?

Reply:
204 is for no content, here we return a success message So if we return 204 I can't return the success message and I need to integrate it somewhere else with a data attribute or something